kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxylicious.java

```diff
@@ -64,13 +64,10 @@ private void deployPortPerBrokerPlain(int replicas) {
     }
 
     /**
-     * Deploy - Port per broker plain with no filters config
-     * @param clusterName the cluster name
-     * @param replicas the replicas
+     * Deploy - Port Identifies Node with no filters config
      */
-    public void deployPortPerBrokerPlainWithNoFilters(String clusterName, int replicas) {
-        createDefaultConfigMap(clusterName);
-        deployPortPerBrokerPlain(replicas);
+    public void deployPortIdentifiesNodeWithNoFilters() {
+        deployKroxyliciousExample(Constants.PATH_TO_OPERATOR_SIMPLE_FILES);
```

Comment:
The Operator examples probably won't live for too long.  We don't want the System Tests to rely on them.  System tests should be in precise control of its own test cases. That said, if you see this a useful short term step, perhaps until #1997 is done, that's fine.

Reply:
Yes, that's was the intention. I'll work on #1997 asap.